src/views/Dashboard.tsx

```diff
@@ -7,6 +7,7 @@ import {
   TotalUsers,
   UsersGraph,
   NewUsersCount,
+  DateTile,
```

Comment:
This is fine for now, but I don't think this `views` folder makes sense, I think this file should go inside `components/Dashboard`